src/tests/Svg.test.ts

```diff
@@ -18,6 +18,9 @@ describe('SVG QRCode class tests', () => {
             correctLevel: QRErrorCorrectLevel.M,
             backgroundDimming: 'rgba(0,0,0,0)',
             logoScale: 0.2,
+            rectangular: false,
+            widthSize: 0,
```

Comment:
@hritikaggarwal31 please use more self-explanatory variable names instead of `widthSize` and `heightSize`